content/docs/reference/state.md

```diff
@@ -1,140 +1,212 @@
 operates, this page gives an overview of the options.
 
 ## Backends
 
-Pulumi supports multiple so-called _backends_ for storing this state. There are two primary kinds:
-
-1. **Web**: a backend accessed by the CLI through REST API calls
-2. **Filesystem**: a backend stored locally on your computer's filesystem
-
-By default, the CLI uses a web backend hosted at https://app.pulumi.com. This is a free service offered by Pulumi
-that also offers advanced tiers for team and enterprise features. It is possible to opt-in to the filesystem backend.
-
-We think that using the web backend and the CLI together provides the right combination of usability, safety,
-and security for most users. But the filesystem backend provides more flexibility for those who need it.
-
-### Web Backend Features
-
-The web backend stores all checkpoint state securely, and the CLI always communicates directly with your cloud provider
-using client-side authentication. All state is encrypted in transit and at rest.
-
-Although backends are primarily responsible for storing state reliably, the web backend also supports additional
-features that the filesystem backend does not. In particular, the web backend supports keeping track of full
+Pulumi supports multiple so-called _backends_ for storing your infrastructure state. By default, the CLI uses a web backend hosted at [app.pulumi.com](https://app.pulumi.com). Pulumi offers this backend as a free service with [advanced tiers](https://www.pulumi.com/pricing/) for team and enterprise features. Using the `pulumi.com` backend and the CLI together provides the right combination of usability, safety,
+and security for most users. But you can opt into the filesystem or cloud storage backend for more flexibility if you need it.
+
+
+<table>
+    <colgroup>
+        <col style="width: 20%">
+        <col style="width: 20%">
+        <col style="width: 35%">
+    </colgroup>
+        <thead>
+                <th>Backend</th>
+                <th>Description</th>
+                <th>Setup</th>
+        </thead>
+        <tbody>
+            <tr>
+                <td><code>pulumi.com</code></td>
+                <td> Accessed by the CLI through REST API calls</td>
+                <td>No additional configuration after <a href="{{< relref "/docs/reference/install" >}}" target="_blank">installing Pulumi.</a></td>
+            </tr>
+            <tr>
+                <td>Self-managed</td>
+                <td>Stored locally on your computer’s filesystem, or remotely using a cloud provider</td>
+                <td>
+                    <ul>
+                        <li>
+                        <b>Filesystem or local</b>: <i>See <a href="{{< relref "/docs/reference/clouds/aws/setup" >}}" target="_blank">AWS Setup</a> and <a href="https://docs.aws.amazon.com/AmazonS3/latest/dev/UsingBucket.html" target="_blank">Working with Amazon S3 buckets</a> for configuration details.</i>
```

Comment:
1. `<b>Local filesyste</b>`
2. why does this link to aws/s3?

Reply:
Fixed this.